app/models/accreditation.rb

```diff
@@ -36,9 +36,16 @@ class Accreditation < ActiveRecord::Base
   ## Case insensitive inclusion validator
   #############################################################################
   def lowercase_inclusion_validator(attribute, collection, blank_ok = true)
-    return if (var = eval(attribute.to_s)).blank? && blank_ok
+    return if (var = self[attribute]).blank? && blank_ok
 
-    if !collection.include?(var.try(:downcase))
+    # if !collection.include?(var.try(:downcase))
+    #   errors.add(attribute, "#{var} not in [#{collection.join(', ')}]")
+    # end
```

Comment:
In case there was an issue following Patricks test - I can remove it, but I normally leave comments like that in as a reference to what I did before.